build.gradle

```diff
@@ -679,6 +679,7 @@ project('spring-integration-jdbc') {
     dependencies {
         api 'org.springframework:spring-jdbc'
         optionalApi "org.postgresql:postgresql:$postgresVersion"
+        optionalApi 'tools.jackson.core:jackson-databind'
```

Comment:
We need to move this to the `testImplementation` as with a `JsonObjectMapper` abstraction there is no strong Jackson requirement anymore.